node/node.go

```diff
@@ -173,7 +173,7 @@ func (n *Node) Start() error {
 		MaxPendingPeers:  n.config.MaxPendingPeers,
 	}
 	running := &p2p.Server{Config: n.serverConfig}
-	log.Info(fmt.Sprint("instance:", n.serverConfig.Name))
+	log.Info("Starting peer-to-peer node", "instance", n.serverConfig.Name)
```

Comment:
It's awkward to see 

```text
INFO [04-05|13:58:27] Starting peer-to-peer node    instance=Geth/v1.6.0-unstable-61f6d812/darwin/go1.8 
...
INFO [04-05|13:58:27] Starting P2P networking 
```

My suggestion is

```text
```text
INFO [04-05|13:58:27] Geth/v1.6.0-unstable-61f6d812/darwin/go1.8 
...
INFO [04-05|13:58:27] Starting P2P networking 
```